backend/report.js

```diff
@@ -211,6 +211,16 @@ exports.restAggregationReport = function(req, res, next) {
         aggregationArray.push({$limit: pageInfo.size});
 
         var aggregationCallback = function(err, groupEntries) {
+            function isTrue(val) {
```

Comment:
Please rewrite all function
```
groupEntries.forEach(function(entry) {
    return entry.readyForApprove = entry.readyForApprove.every(function(readyForApprove) {return readyForApprove;})
}) 
```

This function replaced all green code. Please use anonymous function.